gnomad_qc/v4/resources/basics.py

```diff
@@ -54,19 +54,37 @@ def get_gnomad_v4_vds(
     # Count current number of samples in the VDS
     n_samples = vds.variant_data.count_cols()
 
-    # Obtain withdrawn UKBB samples (includes 5 samples that should be removed from the VDS)
-    excluded_ukbb_samples_ht = hl.import_table(
-        ukbb_excluded_samples_path,
-        no_header=True,
-    ).key_by("f0")
+    # Obtain samples in application 31063 withdraw list
+    ukbb_31063_withdrawn_ht = hl.import_table(ukbb_excluded_31063_path, no_header=True)
+    ukbb_31063_withdrawn_ht = ukbb_31063_withdrawn_ht.key_by(
+        eid_31063=ukbb_31063_withdrawn_ht.f0
+    )
+
+    # Obtain samples in application 26041 withdraw list
+    ukbb_26041_withdrawn_ht = hl.import_table(ukbb_excluded_26041_path, no_header=True)
+    ukbb_26041_withdrawn_ht = ukbb_26041_withdrawn_ht.key_by(
+        eid_26041=ukbb_26041_withdrawn_ht.f0
+    )
+
+    ukbb_application_map_ht = ukbb_application_map.ht()
+    withdrawn_ukbb_samples = ukbb_application_map_ht.filter(
+        hl.literal(ukbb_31063_withdrawn_ht.eid_31063.collect()).contains(
+            ukbb_application_map_ht.eid_31063
+        )
+        | hl.literal(ukbb_26041_withdrawn_ht.eid_26041.collect()).contains(
+            ukbb_application_map_ht.eid_26041
+        )
+    ).s.collect()
 
-    sample_map_ht = ukbb_array_sample_map.ht()
+    sample_map_ht = ukbb_array_sample_map.ht().key_by("ukbb_app_26041_id")
     sample_map_ht = sample_map_ht.annotate(
         withdrawn_consent=hl.is_defined(
-            excluded_ukbb_samples_ht[sample_map_ht.ukbb_app_26041_id]
+            ukbb_26041_withdrawn_ht[sample_map_ht.ukbb_app_26041_id]
         )
     )
-    withdrawn_ids = sample_map_ht.filter(sample_map_ht.withdrawn_consent).s.collect()
+    withdrawn_26041_from_sample_map = sample_map_ht.filter(
```

Comment:
I'm not sure I understand why this is needed?

Reply:
There are some withdrawn samples that only have a matching id in the sample map file and others only in the bridge file
`sample_map = set(sample_map_ht.ukbb_app_26041_id.collect())
bridge = set(mapping_ht_26041_31063.eid_26041.collect())
withdrawn = set(ukb_26041_withdrawn_ht.eid_26041.collect())
print("Number withdrawn samples only in sample map file: " + str(len((withdrawn&sample_map) - (withdrawn&bridge))),
      "\nNumber withdrawn samples only in bridge file: " + str(len((withdrawn&bridge) - (withdrawn&sample_map))))`
      
Number withdrawn samples only in sample map file: 4 
Number withdrawn samples only in bridge file: 3